db/migrations/00025_create_storage_diffs_table.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+CREATE TABLE public.storage_diff
+(
+    id             SERIAL PRIMARY KEY,
+    block_height   BIGINT,
+    block_hash     BYTEA,
```

Comment:
Aren't these strings over `rpc`? :thinking:

Reply:
I think so - hex strings. But I think we'll want to convert those to hashes anyway at some point to assure they're treated uniformly regardless of whether there's a `0x` prefix, and also that storing a `bytea` is cheaper than a `varchar`. Happy to switch up the types if folks think that'd be better, though

Reply:
Nah, whatever you think makes most sense :ok_hand: